packages/client/hmi-client/src/components/model/petrinet/tera-petrinet-tables.vue

```diff
@@ -106,7 +106,7 @@ const states = computed<State[]>(() => props.model?.model?.states ?? []);
 const transitions = computed<Transition[]>(() =>
 	props.model.model.transitions?.map((transition: Transition) => ({
 		...transition,
-		name: transition.name ?? transition?.properties?.name ?? transition.id,
+		name: transition.name ?? transition.properties?.name ?? transition.id,
```

Comment:
Is this correct? You're checking the status of `transition.name` and then return `transition.properties.name` without checking it?

Reply:
`transition.properties` can be `undefined` so there wasn't a need for the first `?`. What's happening here is assigning a name to edit in the UI. The order of these variables are just the priority that I'd assign the default name to be when you'd start editing.

Reply:
err sorry... i was just wondering why we're checking `transition.name` only to then turn around and use `transition.properties?.name`

Reply:
A lot transitions have their `properties.name` filled while the `name` is empty. So starting on that makes sense in a lot of cases.